job-descriptions/software-engineer-frontend.md

```diff
@@ -26,39 +26,39 @@ We are looking for candidates with the following skills and a strong desire to l
 - Ability to communicate clearly and empathetically, especially in writing and documentation.
 - Practiced at creating high quality software balanced with a pragmatic understanding of how to make appropriate tradeoffs (e.g. cut scope) to ship quickly and iterate when necessary.
 
-Nice to have:
+Nice-to-have:
```

Comment:
@nicksnyder - in all the other jobs this is a sub-header of About the role. I assume you made this a non-header on purpose, but wanted to confirm!

Reply:
I was thinking more about content and less about formatting with this. I think it would be nice to be consistent so let's do make this a heading like the other job descriptions.

```suggestion
### Nice-to-haves:
```